### ISSUE TITLE
Software crash after creating new bot over api.

### ISSUE THREAD

**Issue by lightgreenwings** [Splamy/TS3AudioBot#628]
After creating a bot the software crashed :c

**Version**
last develop

**Platform**
linux

**Log**
```
2019-09-03 20:56:57.2366| INFO||WebApi.ProcessApiV1Call ::ffff:46.251.239.26 Requested: /api/settings/create/9_VW9Jb
2019-09-03 20:56:57.2436| INFO||WebApi.ProcessApiV1Call ::ffff:46.251.239.26 Requested: /api/settings/bot/set/9_VW9Jb/connect.name/akzdFM
2019-09-03 20:56:57.2665| INFO||WebApi.ProcessApiV1Call ::ffff:46.251.239.26 Requested: /api/settings/bot/set/9_VW9Jb/connect.channel/%2F
2019-09-03 20:56:57.2774| INFO||WebApi.ProcessApiV1Call ::ffff:46.251.239.26 Requested: /api/settings/bot/set/9_VW9Jb/connect.address/5.181.151.7
2019-09-03 20:56:57.2852| INFO||WebApi.ProcessApiV1Call ::ffff:46.251.239.26 Requested: /api/settings/bot/set/9_VW9Jb/events.onconnect/!x%E1%85%9F(!play%E1%85%9Fhttps:%2F%2Fstream.laut.fm%2Fakzdfm)%E1%85%9F(!repeat%E1%85%9Fone)%E1%85%9F(!bot%E1%85%9Fcommander%E1%85%9Fon)
2019-09-03 20:56:57.2929| INFO||WebApi.ProcessApiV1Call ::ffff:46.251.239.26 Requested: /api/settings/bot/set/9_VW9Jb/events.onidle/!x%E1%85%9F(!play%E1%85%9Fhttps:%2F%2Fstream.laut.fm%2Fakzdfm)%E1%85%9F(!repeat%E1%85%9Fone)%E1%85%9F(!bot%E1%85%9Fcommander%E1%85%9Fon)
2019-09-03 20:56:57.3086| INFO||WebApi.ProcessApiV1Call ::ffff:46.251.239.26 Requested: /api/bot/connect/template/9_VW9Jb
2019-09-03 20:56:57.3086| INFO||Bot.InitializeBot Bot "9_VW9Jb" connecting to "5.181.151.x"
2019-09-03 20:56:57.3764| INFO|48|Ts3Client.UpdateIndentityToSecurityLevel Calculating up to required security level: 12
2019-09-03 20:56:57.4429| INFO|48|Bot.OnBotConnected Bot "9_VW9Jb"(48) connected.
2019-09-03 20:56:57.4429|FATAL|48|Core.ExceptionHandler Critical program failure!
System.NullReferenceException: Object reference not set to an instance of an object
  at TS3Client.Full.Book.Connection.SetChannel (TS3Client.Full.Book.Channel channel, System.UInt64 id) [0x0000d] in <d7be456959874b6d893fd978e4506298>:0 
  at TS3Client.Full.Book.Connection.UpdateChannelList (TS3Client.Messages.ChannelList msg) [0x001c0] in <d7be456959874b6d893fd978e4506298>:0 
  at TS3Client.Full.Ts3FullClient.InvokeEvent (TS3Client.LazyNotification lazyNotification) [0x00712] in <d7be456959874b6d893fd978e4506298>:0 
  at TS3Client.ExtraThreadEventDispatcher.DispatchLoop () [0x0001f] in <d7be456959874b6d893fd978e4506298>:0 
  at TS3Client.ExtraThreadEventDispatcher+<>c__DisplayClass5_0.<Init>b__0 () [0x0000b] in <d7be456959874b6d893fd978e4506298>:0 
  at System.Threading.ThreadHelper.ThreadStart_Context (System.Object state) [0x00014] in <81342d83acda4c5590ec19c7afdf26b9>:0 
  at System.Threading.ExecutionContext.RunInternal (System.Threading.ExecutionContext executionContext, System.Threading.ContextCallback callback, System.Object state, System.Boolean preserveSyncCtx) [0x00071] in <81342d83acda4c5590ec19c7afdf26b9>:0 
  at System.Threading.ExecutionContext.Run (System.Threading.ExecutionContext executionContext, System.Threading.ContextCallback callback, System.Object state, System.Boolean preserveSyncCtx) [0x00000] in <81342d83acda4c5590ec19c7afdf26b9>:0 
  at System.Threading.ExecutionContext.Run (System.Threading.ExecutionContext executionContext, System.Threading.ContextCallback callback, System.Object state) [0x0002b] in <81342d83acda4c5590ec19c7afdf26b9>:0 
  at System.Threading.ThreadHelper.ThreadStart () [0x00008] in <81342d83acda4c5590ec19c7afdf26b9>:0 
2019-09-03 20:56:57.4568| INFO|48|Core.Dispose TS3AudioBot shutting down.
```

**Comment by Splamy** [comment]
Can you pm(mail) me the address, is that consistent?

**Comment by Splamy** [comment]
Fixed on dev (b3d9cbcd5f8d49330fae1aeca587ea0987651a70)
